staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -342,6 +342,8 @@ type ListOptions struct {
 	// +optional
 	ResourceVersion string `json:"resourceVersion,omitempty" protobuf:"bytes,4,opt,name=resourceVersion"`
 	// Timeout for the list/watch call.
+	// For a watch call, this is a limit on the duration of the call,
```

Comment:
suggest "This limits the duration of the call, regardless of any activity or inactivity."

Reply:
Thanks for the suggestion.  Does that apply to `list` as well as `watch`?